src/parser.js

```diff
@@ -0,0 +1,23 @@
+const parseRawLinks = (rawLinks) => {
+  if (typeof rawLinks !== "string") {
+    return [];
+  }
+
+  let links = [];
+  const splitLinks = rawLinks.split(",");
+
+  for (var i = 0; i < splitLinks.length; i++) {
+    let link = {};
+    const splitParts = splitLinks[i].split(";");
+
+    link.url = splitParts[0].replace(/<(.*)>/, "$1").trim();
```

Comment:
The correct terminology for this is `href`. Please use that instead of `url`.

Reply:
Done